### PR TITLE
add doc of conv_emformer

### PR DESCRIPTION
**Pull request by yaozengwei**
This PR adds documents for ConvEmformer.

### CODE REVIEW

File: docs/source/streaming_asr/conv_emformer/client.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. _conv_emformer_client:
+
+Client
+======
+
+With the client you can record you voice in real-time, send it to the
```
Comment (csukuangfj) [inline docs/source/streaming_asr/conv_emformer/client.rst:6]:
```suggestion
With the client you can record your voice in real-time, send it to the
```

**Comment by yaozengwei** [comment]
You can have a review at https://yaozengwei.github.io/sherpa/index.html
Screen Shot 2022-07-21 at 16 07 46

File: docs/source/streaming_asr/conv_emformer/client.rst (hunk 1 above)
Comment (csukuangfj) [reply docs/source/streaming_asr/conv_emformer/client.rst:6]: Please also fix the typo in the other file `client.rst`.

File: docs/source/streaming_asr/conv_emformer/index.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   To be specific, the pre-trained model is trained on the `LibriSpeech`_
+   dataset using the code from
+   `<https://github.com/k2-fsa/icefall/tree/master/egs/librispeech/ASR/conv_emformer_transducer_stateless2>`_.
+
+   The pre-trained model can be downloaded from
+   `<https://huggingface.co/Zengwei/icefall-asr-librispeech-conv-emformer-transducer-stateless2-2022-07-05>`_
+
+There are no **recurrent** modules in the transducer model:
+
+  - The encoder network (i.e., the transcription network) is an ConvEmformer model
```
Comment (csukuangfj) [inline docs/source/streaming_asr/conv_emformer/index.rst:19]:
```suggestion
  - The encoder network (i.e., the transcription network) is a ConvEmformer model
```

**Review by csukuangfj — CHANGES_REQUESTED** [review]

**Review by csukuangfj — APPROVED** [review]

**Pull request merged**